lib/plugins/aws/provider/awsProvider.js

```diff
@@ -325,6 +330,74 @@ class AwsProvider {
         provider: {
           properties: {
             apiGateway: { type: 'object', properties: { websocketApiId: { type: 'string' } } },
+            alb: {
+              type: 'object',
+              properties: {
+                targetGroupPrefix: { type: 'string', maxLength: 16 },
+                authorizers: {
+                  type: 'object',
+                  additionalProperties: {
+                    type: 'object',
+                    properties: {
+                      type: { enum: ['cognito', 'oidc'] },
+                      userPoolArn: { $ref: '#/definitions/awsArnString' },
```

Comment:
I think, even though documentation states _string_, the CloudFormation intrinsic functions are naturally supported. Therefore it should rather be `awsArn` and not `awsArnString`